web-api/src/persistence/postgres/cases/getCaseByDocketNumber.ts

```diff
@@ -16,12 +13,9 @@ export const getCaseByDocketNumber = async ({
   includeConsolidatedCases?: boolean;
   user?: UnknownAuthUser;
 }): Promise<RawCase> => {
-  const [caseData, workItems] = await settlePromises([
-    getCasesByDocketNumbers({
-      docketNumbers: [docketNumber],
-    }),
-    getWorkItemsByDocketNumber({ docketNumber }),
```

Comment:
The only reason we fetched work items here was to attach some information to docket entries. The approach in this PR is to _not_ fetch work items unless we need them, and attach the information we need when we need to.